src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -681,6 +681,7 @@ internal static X509Certificate2Collection BuildBagOfCerts(KeyInfoX509Data keyIn
                         }
                     }
                     catch (CryptographicException) { }
+                    catch (PlatformNotSupportedException) { }
```

Comment:
If it was suffiicently un-obvious that we didn't catch this before, perhaps that suggests a comment is worthwhile...